vector/src/main/scala/geotrellis/vector/GeomFactory.scala

```diff
@@ -16,15 +16,29 @@
 
 package geotrellis.vector
 
+import com.typesafe.config.ConfigFactory
 import com.vividsolutions.jts.geom
 import com.vividsolutions.jts.geom.PrecisionModel
 import com.vividsolutions.jts.precision.GeometryPrecisionReducer
 
+import scala.util.Try
+
 private[vector] object GeomFactory {
 
-  val factory = new geom.GeometryFactory()
+  val precisionType = Try(ConfigFactory.load().getString("geotrellis.jts.precision.type").toLowerCase).getOrElse("floating")
+
+  val precisionModel = precisionType match {
+    case "floating" => new PrecisionModel()
+    case "floating_single" => new PrecisionModel(PrecisionModel.FLOATING_SINGLE)
+    case "fixed" => 
+      val scale = new PrecisionModel(Try(ConfigFactory.load().getDouble("geotrellis.jts.precision.scale")).getOrElse(1e12))
```

Comment:
Mb to add a warn here, that the default value would be used.

Reply:
Also, probably it makes sense to add `precision.scale` and `precision.type` default values into vector project configuration file, that can be a sort of a usage example.

Reply:
Good point, we should provide ‘reference.conf’ with default values. Those will be overwritten by values defined in ‘application.conf’.